examples/source_separation/.gitignore

```diff
@@ -0,0 +1 @@
+data
```

Comment:
Isn't there a toplevel .gitignore? Why have a separate one?

Reply:
data sounds too generic to put under the top-level `.gitignore`, however this is for my dev use and it was not meant to be checked in. I will remove it.